### PR TITLE
Add REDUCE(3f), and miscellaneous errata and style changes

### PR DESCRIPTION
**Pull request by urbanjost**
Added REDUCE description, corrected additional errata and made some style changes, some of which are needed to be able to built the man-pages.

Added
 - /REDUCE.md

Altered
 - /ABS.md
 - /ADJUSTR.md
 - /ALL.md
 - /BGT.md
 - /BIT_SIZE.md
 - /BLE.md
 - /CMPLX.md
 - /CO_REDUCE.md
 - /CPU_TIME.md
 - /DATE_AND_TIME.md
 - /EOSHIFT.md
 - /EXP.md
 - /FINDLOC.md
 - /GET_COMMAND_ARGUMENT.md
 - /GET_ENVIRONMENT_VARIABLE.md
 - /HYPOT.md
 - /IOR.md
 - /MAXLOC.md
 - /MERGE_BITS.md
 - /MIN.md
 - /MOVE_ALLOC.md
 - /NORM2.md
 - /REPEAT.md
 - /SELECTED_REAL_KIND.md
 - /SPREAD.md
 - /SQRT.md
 - /SUM.md
 - /THIS_IMAGE.md
 - /TINY.md
 - /TRANSFER.md

 - index.md
 - transform/index.md

### CODE REVIEW

**Comment by henilp105** [comment]
Thanks sir, @urbanjost .

**Comment by henilp105** [comment]
#build_preview

File: source/learn/intrinsics/_pages/NORM2.md [hunk 1]
```diff
@@ -47,11 +47,10 @@ Sample program:
 ```fortran
 program demo_norm2
 implicit none
-integer :: i
 
 real :: x(3,3) = reshape([ &
    1, 2, 3, &
-   4, 5, 6, &
+   4, 5 ,6, &
```
Comment (awvwgk) [inline source/learn/intrinsics/_pages/NORM2.md:53]: Is this an intended change?

**Review by awvwgk — CHANGES_REQUESTED** [review]
Looks like this patch reverts parts of https://github.com/fortran-lang/webpage/pull/142 for whitespace around punctuation.

**Comment by henilp105** [comment]
#build_preview

File: source/learn/intrinsics/_pages/NORM2.md (hunk 1 above)
Comment (urbanjost) [reply source/learn/intrinsics/_pages/NORM2.md]: No. Ships passing in the sea, I'm afraid. I incorporated #146 changes that affected the intrinsic descriptions.

**Review by awvwgk — APPROVED** [review]
Thanks for sharing. Looks good to me now.

**Comment by henilp105** [comment]
#delete_preview

**Pull request merged**